frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -4378,30 +4378,46 @@ const TypedFnSignature* tryResolveInitEq(Context* context,
   return c.mostSpecific().only().fn();
 }
 
-const TypedFnSignature* tryResolveDeinit(Context* context,
-                                         const AstNode* astForScopeOrErr,
-                                         const types::Type* t,
-                                         const PoiScope* poiScope) {
+static const TypedFnSignature*
+tryResolveZeroArgMethod(Context* context, UniqueString name,
+                        const AstNode* astForScopeOrErr,
```

Comment:
To make the query using this more reusable, let's change `astForScopeOrErr` into an `ID` since that is all that is needed within